### PR TITLE
Fix/compatibility note

### PR DESCRIPTION
**Pull request by glefloch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,10 +26,15 @@ If you are using gradle:
 
 ````groovy
 dependencies {
-    testImplementation 'io.quarkiverse.mockk:quarkus-junit5-mockk:0.3.0'
+    testImplementation 'io.quarkiverse.mockk:quarkus-junit5-mockk:LATEST'
 }
 ````
 
+## Compatibility with quarkus
```
Comment (gsmet) [inline README.md:33]:
```suggestion
## Compatibility with Quarkus
```

File: README.md [hunk 2]
```diff
@@ -26,10 +26,15 @@ If you are using gradle:
...
+
+Starting from quarkus `2.8` and above, version `1.1.0` of `quarkus-junit5-mockk` should be use.
```
Comment (gsmet) [inline README.md:35]:
```suggestion
Starting with Quarkus `2.8` and above, version `1.1.0` of `quarkus-junit5-mockk` should be used. 
```

File: README.md [hunk 3]
```diff
@@ -26,10 +26,15 @@ If you are using gradle:
...
 dependencies {
-    testImplementation 'io.quarkiverse.mockk:quarkus-junit5-mockk:0.3.0'
+    testImplementation 'io.quarkiverse.mockk:quarkus-junit5-mockk:LATEST'
 }
 ````
 
+## Compatibility with quarkus
+
+Starting from quarkus `2.8` and above, version `1.1.0` of `quarkus-junit5-mockk` should be use. 
+If you use a version before `2.7`, version `1.0.1` of `quarkus-junit5-mockk` should be use.
```
Comment (gsmet) [inline README.md:36]:
```suggestion
If you use a version before `2.7`, version `1.0.1` of `quarkus-junit5-mockk` should be used.
```

File: docs/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -32,6 +32,11 @@ The extension exposes two annotations:
 * `@InjectMock` to inject MockK mocks in your application
 * `@InjectSpy` to inject SpyK spies in your application
 
+== Compatibility with quarkus
+
+Starting from quarkus `2.8` and above, version `1.1.0` of `quarkus-junit5-mockk` should be use.
+If you use a version before `2.7`, version `1.0.1` of `quarkus-junit5-mockk` should be use.
```
Comment (gsmet) [inline docs/modules/ROOT/pages/index.adoc:38]:
```suggestion
== Compatibility with Quarkus

Starting with Quarkus `2.8` and above, version `1.1.0` of `quarkus-junit5-mockk` should be used.
If you use a version before `2.7`, version `1.0.1` of `quarkus-junit5-mockk` should be used.
```

**Review by gsmet — CHANGES_REQUESTED** [review]

**Comment by glefloch** [comment]
Thanks for the review @gsmet I pushed fixes

**Review by gsmet — APPROVED** [review]

**Pull request merged**